autoload/asyncomplete/sources/flow.vim

```diff
@@ -66,6 +70,21 @@ function! asyncomplete#sources#flow#get_source_options(opts)
        \ })
 endfunction
 
+let s:flowbin_path_by_dir = {} " dir: <path to flow>
+
+function! s:resolve_flowbin_path(file, fallback)
+    let l:dir = fnamemodify(a:file, ':h')
+    if !has_key(s:flowbin_path_by_dir, l:dir)
+        let l:node_dir = asyncomplete#utils#find_nearest_directory(a:file, 'node_modules')
+        if filereadable(l:node_dir . '/.bin/flow')
```

Comment:
Just to be safe, if the length of `s:flowbin_path_by_dir` is 20, let us empty it. LRU cache would be too much for now. I usually have my vim open for days and this would increase the memory as I open more files.